zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -85,6 +86,7 @@
   public static Server jettyWebServer;
   public static NotebookServer notebookWsServer;
   public static Helium helium;
+  public static UsersRepo usersRepo;
```

Comment:
+1 for `TODO` with the variable name for the current state. @tinkoff-dwh and few of community would know the history why this variable name is `userRepo`? But I think the others wouldn't